web/src/helpers/server/get-first-app/index.ts

```diff
@@ -0,0 +1,17 @@
+import { getAPIServiceGraphqlClient } from "@/backend/graphql";
+import { getSdk } from "./graphql/getAppsByUserAndTeams.generated";
+
+export const getFirstApp = async (params: {
```

Comment:
what's the rationale for requiring here the `user_id` as well? wouldn't you fetch specifically for a team only and if you're tampering with the team_id Hasura permissions would catch it

Reply:
You are right

Reply:
Updated